how-to/editors-guide.md

```diff
@@ -120,7 +120,7 @@ All reviews happen in GitHub issues. The template for the
 `yaml` header of a review submission below will be
 referenced multiple times in the steps below:
 
-```{include} ../appendices/issue-review-template.md
+```{include}
```

Comment:
Hey @crhea93, I'm curious about what removing these resources includes? They should render the content on the page for the user to copy. Was this intentional?

Reply:
Weirdly, it's rendering in CI, but I don't understand how because the path to the include is removed? 🙃 i will come back to this later unless I am just missing some magic!

Reply:
Ah no it should not be removed!! Let me look into why it is doing this....